Sso/config/services.php

```diff
@@ -35,12 +35,19 @@
         'secret' => env('STRIPE_SECRET'),
     ],
 
-    'google' => [
-        'client_id' => env('GOOGLE_CLIENT_ID', 'your-github-app-id'),
-        'client_secret' => env('GOOGLE_CLIENT_SECRET', 'your-github-app-secret'),
-        'redirect' => env('GOOGLE_CALLBACK_URL', 'http://your-callback-url'),
+    'auth' => [
+        'google' => [
+            'name' => 'Google',
+            'allow' => false,
+            'client_id' => env('GOOGLE_CLIENT_ID', 'your-github-app-id'),
+            'client_secret' => env('GOOGLE_CLIENT_SECRET', 'your-github-app-secret'),
+            'redirect' => env('GOOGLE_CALLBACK_URL', 'http://your-callback-url'),
+            'is_default' => true
```

Comment:
I'll be extracting `is_default` to the separate `default_auth` config option read from the `env`. By default, `google` will remain as default, but anyone will be able to specify the provider from within the `.env` file.